puffin_egui/src/flamegraph.rs

```diff
@@ -962,7 +962,7 @@ fn paint_thread_info(info: &Info<'_>, thread: &ThreadInfo, pos: Pos2, collapsed:
     };
 
     info.painter.rect_filled(rect.expand(2.0), 0.0, back_color);
-    info.painter.galley_with_color(rect.min, galley, text_color);
+    info.painter.galley(rect.min, galley, text_color);
```

Comment:
You should pass this `text_color` to `layout_no_wrap` above, or the color won't take.

Reply:
The color passed to `layout_no_wrap` (`Rgba::from_white_alpha(0.9)`) used to always be overridden here.

Let's avoid `galley_with_override_text_color` because it has a long complicated name (for a good reason: I want to discourage its use).

There are two solutions: either use `egui::Color32::PLACEHOLDER` as the color passed to `layout_no_wrap`, or pass the same` text_color` to both `layout_no_wrap` and `galley`. I suggest the latter.

Reply:
`text_color` depends on the output from the `layout_no_wrap` call, so I had to go for the `PLACEHOLDER` route.